Sources/RichTextKit/Actions/RichTextAction.swift

```diff
@@ -149,11 +132,15 @@ public extension RichTextAction {
         case .stepSuperscript(let steps): .actionStepSuperscript(steps)
         case .toggleStyle(let style): style.titleKey
         case .undoLatestChange: .actionUndoLatestChange
-        case .setBackgroundColor: .backgroundColor
-        case .setForegroundColor: .foregroundColor
-        case .setUnderlineColor: .underlineColor
-        case .setStrikethroughColor: .strikethroughColor
-        case .setStrokeColor: .strokeColor
+        case .setColor(_, let richTextColor):
```

Comment:
@DominikBucher12 I thiiiiink the rich text color has a title that you can return right away.

Reply:
It actually does not :( I wouldn't implement it in there IMHO, but the icons can stay.

Reply:
I'll fix! :)